Improve Telemetry Doc. (#208)
omichel/webots: docs/guide/telemetry.md

| disableShadows            | The shadows option set in the [preferences](preferences.md#opengl)                                                                                              | false                          |
| GTAO                      | The ambient occlusion option set in the [preferences](preferences.md#opengl)                                                                                    | 2                              |
| SMAA                      | The main 3D view anti-aliasing option set in the [preferences](preferences.md#opengl)                                                                           | true                           |

> According to our will to be as transparent as possible we do provide all the raw data [here](https://cyberbotics.com/telemetry).

Reviewer: ```suggestion
To be as transparent as possible, all the telemetry raw data is available [here](https://cyberbotics.com/telemetry).
```
